### ISSUE TITLE
Question about training data labeling

### ISSUE THREAD

**Issue by tanggang1997** [facebookresearch/audiocraft#229]
Now, I tried to label the data in Chinese, but after 50 epochs, I found that the generated audio was very current sounding.
